src/js/organisms/filter-form.js

```diff
@@ -89,7 +89,15 @@ module.exports = class FilterForm {
     });
 
     if (matchedCheckboxes.length) {
-      this.form.dispatchEvent(new Event('input'));
+      let event;
+      if (typeof(Event) === 'function') {
```

Comment:
`typeof` is a keyword so you don’t need the parentheses around `Event`. `typeof Event === 'function'` works the same.

Could this be shimmed like the other methods in pollyfills.js? Something like:

```
if (!window.Event) {
  window.Event = function(typeArg) {
    const event = document.createEvent('Event');
    event.initEvent(typeArg, true, true);
    return event;
  };
}
```

I think that should work for this case, but [proper pollyfills are available][1] too.

[1]: https://github.com/webcomponents/polyfills/tree/master/packages/webcomponentsjs

Reply:
Added a shim as suggested.